examples/circuit-relaying/README.md

```diff
@@ -0,0 +1,336 @@
+Gateway (readonly) is listening on: /ip4/127.0.0.1/tcp/9090
+Daemon is ready
+```
+
+Look out for an address similar to `/ip4/127.0.0.1/tcp/4003/ws/ipfs/Qm...`. Note it down somewhere, and let's move on to the next step. 
+
+### 2. Configure and run the bundled example
+
+Now that we have ipfs installed and initialized, let's set up the included example. This is a standard npm package, so the usual `npm install` should get us going. Let's `cd` into the `examples/circuit-relaying` directory and run:
+
+```
+npm install
+```
+
+After it finishes, we should be able to run the project with `npm start` and get output similar to:
+
+```
+npm run start
+Server running at http://localhost:1234
+```
+
+The bundled example is a simple chat app that uses another cool ipfs feature - [pubsub](https://github.com/libp2p/specs/tree/master/pubsub). Let's open up a browser and paste the above url into the address bar. We should see something similar to the following image:
+
+![](./img/img1.png)
+
+### 3. Connect the two browser nodes to the circuit relay
+
+In order for our browser nodes to be able to messages each other, we need to get them connected. But do do that, we need to use a relay - browser nodes can't be connected directly because of lack of socket support. 
+
+Remember the caveat above `Currently a Relay will only work if it already has a connection to the STOP node`? This means that we need to connect our browser nodes to the relay node first.
+
+Having both browsers running side by side (as shown in the first screenshot), enter the `/ip4/127.0.0.1/tcp/4003/ws/ipfs/...` address noted above into the `Connect to Peer` field and hit the `Connect` button:
+
+![](./img/img3.png)
+
+After connecting to the IPFS node, we should see the relay peer show up under the `Peers Connected` box.
+
+![](./img/img4.png)
+
+Let's repeat the same steps with the second tab. Now, both of our browser nodes should be connected to the relay and we can move on to the next step.
+
+### 4. Dial the two browser nodes using a `/p2p-circuit` address
+
+Now that both browsers are running side by side (as shown in the first screenshot), let's get them connected to each other. Head out to the `Addresses` box in one of the tabs, copy the `/p2p-circuit` address and then paste it into the `Connect to Peer` box in the other tab. Repeat these steps on the second tab.
+
+![](./img/img5.png)
+
+Let's hit the `Connect` button on each of the tabs and we should get the two browsers connected and join the chat room.
+
+![](./img/img6.png)
+
+### 5. Send data browser to browser.
+
+Now that we have the two browsers connected, let's try the app out. Type a few words in one of the browser windows and you should see them appear in the other as well!
+
+![](./img/img7.png)
+
+Thats it!
+
+### So what just happened?
+
+Good question! 
+
+- We used [js-ipfs](htpps://github.com/ipfs/js-ipfs) running in the browser with circuit relay enabled:
+  - _Notice the `EXPERIMENTAL.relay.enabled` bellow_
```

Comment:
Typo: “bellow” → “below”

We should probably also call out `EXPERIMENTAL.pubsub`, since that’s also a non-default. (right?)

Reply:
ack on typo. 

I think we can leave `pubsub` out (remove from options in the example), cause it doesn't have much to do with circuit - I think it would add to the confusion?